org-repo-migrator.sh

```diff
@@ -104,55 +184,62 @@ main()
-  check=$(curl -s ${URL}/${VERSION}/repositories/${src}/ | jq  '.results[]|"\(.name)=\(.is_private)"') > /dev/null
-	
-  # Loop over the number of repositories in source organization 
-  for i  in ${check}
+ # Loop to iterate on the number of repositories in every page
+  for (( repo_page=1;;repo_page++ ));
   do
-    # Fetch the name of the repository
-    name=$(echo ${i} | sed -e 's/\"//g' -e 's/=.*//')
-
-    # If condition to check whether the repository is to be skipped
-    if [[ ! "${skip_repos[@]}" =~ "${name}" ]]; then
-         
-      # Fetch the repository privacy whether public/private repository    
-      repo_visibility=$(echo $i | sed -e 's/\"//g' -e 's/.*=//g')
-         
-      # Fetch the image tags for the repos
-      image_tags=$(curl -s ${URL}/${VERSION}/repositories/${src}/${name}/tags/ | jq -r '.results|.[]|.name') > /dev/null 
-   
-        # Loop to fetch a tag from source org repos and apply to the destination org repos 	   
-        for tag in ${image_tags}
-        do	
-          # Check whether the repo is public/private repository	    
-          if [[ "${repo_visibility}" = "${visibility}" ]]; then
-            echo "Pulling ${name} with tag ${tag} from source ${src}"
-            # Pulling repository from the source organzation    
-            docker pull ${src}/${name}:${tag} > /dev/null
-
-            echo "Pulling repository ${name}:${tag} successful" 
-            echo "Tagging the repository from ${src}/${name}:${tag} to ${dest}/${name}:${tag}"
-            # Tagging a repository with tag to to destination org with tag
-            docker tag ${src}/${name}:${tag} ${dest}/${name}:${tag} > /dev/null
-
-            echo "Repository ${name}:${tag} tagged successfully"
-            echo "Pushing to ${dest} organization the ${name}:${tag} repository"
-            # Pushing the repository to destination org with specific tag
-            docker push ${dest}/${name}:${tag} > /dev/null
-
-            echo "Push successful for ${name}:${tag}"
-          else
-            # If repo is a private repository, skip the execution   
-            continue
-          fi
-        done
-    else
-      # Skip current repository being added in skip_repos variable
+    # Function to fetch the repositories in every page
+    fetchRepos "${URL}" "${VERSION}" "${src}" "${repo_page}" "${size_of_page}" list_of_repos repo_names
+    # Loop over the number of repositories in source organization 
+    for i  in $repo_names
+    do
+      # Fetch the name of the repository
+      name=$(echo ${i} | sed -e 's/\"//g' -e 's/=.*//')
+      # If condition to check whether the repository is to be skipped
+      if [[ ! "${skip_repos[@]}" =~ "${name}" ]]; then
+        # Fetch the repository privacy whether public/private repository    
+        repo_visibility=$(echo $i | sed -e 's/\"//g' -e 's/.*=//g')
+        # Fetch the image tags for the repos
+	for (( tag_page=1;;tag_page++ ));
```

Comment:
```suggestion
	  for (( tag_page=1;;tag_page++ ));
```
Fix indentation for complete if block. Shift it right by 2 spaces